about.html

```diff
@@ -0,0 +1,97 @@
+            <ul class="mainMenu">
+                <a href="#" class="nav-logo">Tonny's Farm</a>
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+                <li>Program</li>
+                <li>Join</li>
+                <li>Sponsor</li>
+                <li>News</li>
+                <li id="border-line">Campaign</li>
+                <li class="closeMenu"><i class="fa fa-times"></i></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="landing">
+        <div class="about-landing-content">
+            <h1 class="heading1 about-none">"Always Fresh Everyday" </h1>
+            <h2 class="heading2">CREATIVE Agriculture <br>GLOBAL SUMMIT 2022</h2>
+            <p class="about-big">We want you to know exactly who is growing your food. You’re welcome to visit the farms and see the love they put into growing your food.</p>
+            <p class="about-small">Please contact us per Email any further questions about CREATIVE Agriculture Global Summit 2022</p>
+            <p class="ccsummit"><b>tonykanyingah@gmail.com</b></p>
+        </div>
+    </section>
+    <section class="about-page1">
+        <div class="prg1">
+            <h3 class="about-program">CREATIVE Agriculture Global Summit 2022</h3>
+            <hr class="underline">
+        </div>
+        <div>
+            <p class="about-big">Improving Farmers’ Livelihood Slowly but sure, by cutting the complex supply chain and food system, we hope to improve the welfare of farmers by giving them the returns they deserve for their hard work.</p>
+        </div>
+        <img class="tonnys-logo" src="./images/kn (1).png" alt="logo about-content">
+    </section>
+
+    <section class="about-page2">
+        <div class="prg1">
+            <h3 class="about-program">See the past Creative Agriculture Global Summits</h3>
+            <hr class="underline">
+        </div>
+        <div class="about-images">
+            <div class="imageleft">
+                <img src="./images/image1.jpg" class="photo" alt="Confrence 2021">
+            </div>
+            <div class="overlay">
+                <p class="overlay-text">Conference 2022</p>
+            </div>
+            <div class="imageleft">
+                <img src="./images/image2.jpg" class="photo" alt="Conference 2022">
+            </div>
+            <div class="overlay">
+                <p class="overlay-text">Conference 2022</p>
+            </div>
+        </div>
+    </section>
+    <div class="footer-class2">
+        <div class="prg1">
+            <h3 class="program">Partners</h3>
+            <hr class="underline">
+            <a class="bold">mozilla</a>
+            <a class="faint">Google</a>
+            <a class="bold">NAVER</a>
+            <a class="faint">daumkakao</a>
+            <a class="bold"><i class="fa fa-airbnb"></i>airbnb</a>
+        </div>
+    </div>
```

Comment:
- [ ] **[OPTIONAL]** Please 🙏🏽 use `footer` tag instead of `div` tag for the best practices of HTML & CSS.
- [ ] **[OPTIONAL]** Kindly add the corresponding footer in the about section to look similar to the original design. Be careful with the differences between the desk version and the mobile version. 👇🏽 

**Original design**

![image](https://user-images.githubusercontent.com/58642949/190834214-56c717f0-3f90-4065-acb8-7ee7d1fe70c8.png)

![image](https://user-images.githubusercontent.com/58642949/190834482-42d3a9be-b22d-42b4-b887-2e29e01b3fca.png)


**Your design** 

![image](https://user-images.githubusercontent.com/58642949/190834264-d7a652ce-76f7-4255-8020-a0e84352f0e2.png)